tests/end_to_end/candid_rpc/class_syntax/complex_init/test/test.ts

```diff
@@ -25,4 +30,4 @@ const recInitCanister = createRecActor(getCanisterId('rec_init'), {
 runTests(() => {
     describe('complex init canister', getTests(complexInitCanister));
     describe('rec init canister', getRecTests(recInitCanister));
-});
+}, complexInitCanisterName);
```

Comment:
I thought you got multiple canisters working?

Reply:
I did... apparently I forgot to go apply it... I'm not sure how careful your review was if you didn't notice it in here ;)

Reply:
done